src/components/CredentialsStep.tsx

```
@@ -0,0 +1,189 @@
+      if (err instanceof DOMException && err.name === "AbortError") return;
+      setStatus({
+        type: "error",
+        message: `Failed: ${err instanceof Error ? err.message : err}`,
+      });
+    } finally {
+      if (!controller.signal.aborted) setSaving(false);
+    }
+  };
+
+  const EyeOpen = (
+    <svg width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"><path d="M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z"/><circle cx="12" cy="12" r="3"/></svg>
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)